CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ This name should be decided amongst the team before the release.
 - [#785](https://github.com/tweag/topiary/pull/785) Added the `coverage` command, that checks how much of the query file is used by the input.
 - [#786](https://github.com/tweag/topiary/pull/786) Re-introduce tests to check that all of the language queries are useful.
 - [#747](https://github.com/tweag/topiary/pull/747) Added support for specifying paths to prebuilt grammars in Topiary's configuration
+- [#832](https://github.com/tweag/topiary/pull/832) Added a `typos.toml` configuration file for `typos-cli` spellchecking
```

Comment:
If you move the configuration to `Cargo.toml`, this will need tweaking. Plus, don't forget to credit yourself :pray: